docs/deployment/deploy-without-docker/Deploying.md

```diff
@@ -1,131 +1,29 @@
-
-```bash
-java \
-    -jar \
-    -Dauthenticate=noauthsessionservice \
-    portal/target/dependency/webapp-runner.jar \
-    portal/target/cbioportal.war
-```
-
-### Optional login
-
-In this mode users can see all the data in the portal, but to save their own groups and virtual studies they are required to log in. This will allow them to store user data in the session service. See the [tutorials](https://www.cbioportal.org/tutorials) section to read more about these features.
+In this mode users are able to use the portal, but they won't be able to save their own virtual studies and groups.
 
-```bash
-java \
-    -jar \
-    -Dauthenticate=social_auth_google,social_auth_microsoft \
-    portal/target/dependency/webapp-runner.jar \
-    portal/target/cbioportal.war
 ```
-
-Google and Microsoft live are supported as optional login currently. Possible values for authenticate are
-
-```bash
--Dauthenticate=social_auth_google,social_auth_microsoft
--Dauthenticate=social_auth_google
--Dauthenticate=social_auth_microsoft
+java -jar target/cbioportal-exec.jar -Dauthenticate=false
 ```
 
-One needs to set the Google/Microsoft related configurations in the `application.properties` file:
-
-```
-#For Google
-googleplus.consumer.key=
-googleplus.consumer.secret=
-
-#For Microsoft
-microsoftlive.consumer.key=
-microsoftlive.consumer.secret=
-```
-
-See [Google's Sign in Documentation](https://developers.google.com/identity/sign-in/web/sign-in#before\_you\_begin) to obtain these values.
-
-See [Microsoft Sign in Documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-web-app-sign-user-app-registration) to obtain these values.
-
-### Required login
-
-```bash
-java \
-    -Dauthenticate=CHOOSE_DESIRED_AUTHENTICATION_METHOD \
-    -jar \
-    portal/target/dependency/webapp-runner.jar \
-    portal/target/cbioportal.war
-```
-
-Change `CHOOSE_DESIRED_AUTHENTICATION_METHOD` to one of `googleplus`, `saml`, `openid`, `ad`, `ldap`. The various methods of authentication are described in the [Authorization and Authentication](/deployment/authorization-and-authentication) section.
-
-### Property configuration
-
-The configuration defined in `application.properties` can also be passed as command line arguments. The priority of property loading is as follows:
-
-1. `-D` command line parameters overrides all
-2. `${PORTAL_HOME}/application.properties`
-3. `application.properties` supplied at compile time
-4. Defaults defined in code
-
-Note that the `authenticate` property is currently required to be set as a command line argument, it won't work when set in `application.properties` (See issue [#6109](https://github.com/cBioPortal/cbioportal/issues/6109)).
-
-Some scripts require a `${PORTAL_HOME}/application.properties` file, so it is best to define the properties there.
```

Comment:
maybe still good to explain a bit how property loading works in java spring boot. Or provides some link to where this is explained? I think good to assume people are not Java devs or junior Java devs

Reply:
It's maybe not specific to building from source, so it could go somewhere else too. But then maybe still good to link to it from here somewhere